src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -49,12 +50,12 @@
     private EnvelopeCreator() {
     }
 
-    public static InputStream getMetaFile() {
-        return EnvelopeCreator.class.getResourceAsStream("/metafiles/valid/from-spec.json");
+    private static InputStream getMetaFile(String fileName) {
+        return EnvelopeCreator.class.getResourceAsStream(METAFILE_BASE_PATH + fileName);
```

Comment:
How about providing a full path to the resource in the call? This would make it a bit clearer what resource we're referencing, as the path provides some information.